packages/create-tldraw/src/group-select.ts

```diff
@@ -0,0 +1,148 @@
+/*!
```

Comment:
hm, checking here, are these not useful?
- https://github.com/bombshell-dev/clack/blob/1adb270af5509ead3d22c4fa602b465f39c7a8c2/packages/prompts/src/group.ts
- or, https://github.com/bombshell-dev/clack/blob/1adb270af5509ead3d22c4fa602b465f39c7a8c2/packages/prompts/src/group-multi-select.ts

Reply:
sadly not